catroid/src/main/java/org/catrobat/catroid/ui/WebViewActivity.java

```diff
@@ -199,6 +199,11 @@ public boolean shouldOverrideUrlLoading(WebView view, String url) {
 		public void onReceivedError(WebView view, int errorCode, String description, String failingUrl) {
 			if (Utils.checkIsNetworkAvailableAndShowErrorMessage(WebViewActivity.this)) {
 				ToastUtil.showError(getBaseContext(), R.string.error_unknown_error);
+			} else {
+				setContentView(R.layout.activity_network_error);
+				setSupportActionBar(findViewById(R.id.toolbar));
+				getSupportActionBar().setIcon(R.drawable.pc_toolbar_icon);
+				getSupportActionBar().setTitle(R.string.app_name);
 			}
```

Comment:
in short, please change the body of this callback to suggested :
```suggestion
			if (errorCode == ERROR_HOST_LOOKUP || description.contains("ERR_INTERNET_DISCONNECTED")) {
				setContentView(R.layout.activity_network_error);
				setSupportActionBar(findViewById(R.id.toolbar));
				getSupportActionBar().setIcon(R.drawable.pc_toolbar_icon);
				getSupportActionBar().setTitle(R.string.app_name);
			} else {
				Log.e(TAG, "could'nt connect to the server! info: " + description + " : " + errorCode);
			}
```